mmv1/templates/terraform/post_create/datastream_stream.go.erb

```diff
@@ -0,0 +1,17 @@
+<%- # the license inside this block applies to this file
+  # Copyright 2022 Google Inc.
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+-%>
+if err := waitForDatastreamStreamReady(d, config, d.Timeout(schema.TimeoutCreate) - time.Minute); err != nil {
```

Comment:
Should it be handled already, as the API call returns operation instead of the resource? I've never tested, so I could be totally wrong

Reply:
@iperetz-goo do we need to worry about the Create call ever returning a non-stable state like `STARTING`?